pack/fhv/fhvc.json

```diff
@@ -104,9 +104,359 @@
+    "traits": "Ruins.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "The endless stretch of ash appears to be completely devoid of life.",
+    "clues": 1,
+    "code": "10557",
+    "double_sided": true,
+    "encounter_code": "the_silent_heath",
+    "encounter_position": 9,
+    "faction_code": "mythos",
+    "flavor": "The soft ash makes navigating the steep terrain especially treacherous.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Ashen Slope",
+    "pack_code": "fhvc",
+    "position": 57,
+    "quantity": 1,
+    "shroud": 4,
+    "text": "<b>Forced</b> - When you reveal Ashen Slope: Test [agility] (4). If you fail, take 2 damage.",
+    "traits": "Blight.",
+    "type_code": "location"
+  },
+  {
+    "clues": 3,
+    "code": "10558",
+    "encounter_code": "the_silent_heath",
+    "encounter_position": 10,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
+    "name": "Salt Chamber",
+    "pack_code": "fhvc",
+    "position": 58,
+    "quantity": 1,
+    "shroud": 4,
+    "text": "<b>Revelation</b> - Put Salt Chamber into play adjacent to the location from which it was drawn\n<b>([codex] Ω)</b> <i>(page 21)</i>.\n[action] Investigators at Salt Chamber spend 1 [per_investigator] clues, as a group: Draw the top card of the stack beneath Salt Chamber. (Group limit once per round.)",
+    "traits": "Cave. Lair.",
+    "type_code": "location"
+  },
+  {
+    "encounter_code": "the_silent_heath",
+    "pack_code": "fhvc",
+    "code": "10559",
+    "position": 59,
+    "encounter_position": 11,
+    "clues": 3,
+    "faction_code": "mythos",
+    "illustrator": "BalanceSheet",
+    "name": "Larval Tunnel",
+    "quantity": 1,
+    "shroud": 2,
+    "text": "<b>Revelation</b> - Put Larval Tunnel into play adjacent to the location from which it was drawn. Search the encounter deck and discard pile for a Colorless Larva and spawn it at this location <b>([codex] Ψ)</b> <i>(page 21)</i>.\n[reaction] After you successfully evade an enemy at Larval Tunnel: Draw the top card of the stack beneath this location. (Group limit once per round.)",
+    "traits": "Cave. Lair.",
+    "type_code": "location"
+  },
+  {
+    "encounter_code": "the_silent_heath",
+    "pack_code": "fhvc",
+    "code": "10560",
+    "position": 60,
+    "encounter_position": 12,
+    "clues": 2,
+    "faction_code": "mythos",
+    "illustrator": "BalanceSheet",
+    "name": "Crystal Nursery",
+    "quantity": 1,
+    "shroud": 3,
+    "text": "<b>Revelation</b> - Put Crystal Nursery into play adjacent to the location from which it was drawn <b>([codex] Ф)</b> <i>(page 21)</i>.\n[reaction] After you end your turn in Crystal Nursery, take 1 direct horror: Draw the top card of the stack beneath Crystal Nursery. (Group limit once per round.)",
+    "traits": "Cave. Lair. Blight.",
+    "type_code": "location"
+  },
+  {
+    "encounter_code": "the_silent_heath",
```

Comment:
Ah, I see throughout you've adopted a slightly different ordering of the lines. The preference is for alphabetical - it's easier for proofing to spot if something's missing. But I won't keep commenting to ask you to change it!

Reply:
No it's fine, I forgot to sort these.